ops/terraform/modules/app_gateway/main.tf

```diff
@@ -142,33 +187,48 @@ resource "azurerm_application_gateway" "load_balancer" {
   }
 
   request_routing_rule {
-    name                       = local.request_routing_rule_name_api
+    name                       = local.request_routing_rule_name_api_ocr
     priority                   = 100
     rule_type                  = "Basic"
-    http_listener_name         = local.listener_name_api
-    backend_address_pool_name  = local.backend_address_pool_name_api
-    backend_http_settings_name = local.http_setting_name_api
+    http_listener_name         = local.listener_name_api_ocr
+    backend_address_pool_name  = local.backend_address_pool_name_api_ocr
+    backend_http_settings_name = local.http_setting_name_api_ocr
   }
 
+  request_routing_rule {
+    name                       = local.request_routing_rule_name_api_middleware
+    priority                   = 150
+    rule_type                  = "Basic"
+    http_listener_name         = local.listener_name_api_middleware
+    backend_address_pool_name  = local.backend_address_pool_name_api_middleware
+    backend_http_settings_name = local.http_setting_name_api_middleware
+  }
 
   url_path_map {
     name                               = "${var.name}-${var.env}-urlmap"
     default_backend_address_pool_name  = local.backend_address_pool_name_static
     default_backend_http_settings_name = local.http_setting_name_static
-    default_rewrite_rule_set_name      = "${var.name}-routing"
+    default_rewrite_rule_set_name      = "${var.name}-middleware-routing"
 
     path_rule {
-      name                       = "api"
+      name                       = "ocr"
```

Comment:
Is the name just `ocr` or `api_ocr`?

Reply:
oh the name here doesn't really matter, just needs to be different than the middleware rule.